### ISSUE TITLE
Logging out doesn't close "Editor"

### ISSUE THREAD

**Issue by PeterJCLaw** [redox-os/redox#805]
**Reproduction**:
* Boot the 0.0.5 iso image
* Click the file browser from the task bar at the bottom
* Click the README.md file in the launched file browser (which launches it in the Editor)
* Click Logout in the bottom right

**Expected behavior**:
When logging out, all open programs should close.

**Actual behavior**:
All other programs close as expected, though the Editor instance remains live and can be interacted with even while the Orbital login dialog is shown.

![editor-over-login](https://cloud.githubusercontent.com/assets/336212/21593984/5403ff8e-d115-11e6-9163-9d2a427a5a93.png)

**Misc**:
I'm using VirtualBox 5.1.12 on Ubuntu 16.04 and had to boot without a network card in order to get Orbital to show up. I'm assuming that the no-network-card issue is probably unrelated though.

**Comment by jackpot51** [comment]
Pretty interesting - how did you launch the editor? Was it through the file manager?

The logout button should send a kill signal to all child processes - it probably did not signal the editor.

**Comment by PeterJCLaw** [comment]
Yeah, the editor opens when I click on "README.md" in the file browser.
I imagine the editor ends up as a grandchild process (with the file browser as the intermediate parent).

If I just launch the editor directly then it does get closed on logout.

**Comment by jackpot51** [comment]
Yes, that appears to be what is happening. The grandchildren are not getting the signal.

Thanks for thoroughly testing!

**Comment by bjorn3** [comment]
Maybe use a different orbital server for every user, so that processes of another user/session dont show up on your session.
